<ui version="4.0" >
 <class>ContentClass</class>
 <widget class="QWidget" name="ContentClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Content</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QGroupBox" name="groupBox_HDD_USB" >
     <property name="font" >
      <font>
       <family>Book Antiqua</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="title" >
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" >
            <item>
             <widget class="QLabel" name="label_HDD" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="font" >
               <font>
                <family>Book Antiqua</family>
                <pointsize>18</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="pixmap" >
               <pixmap resource="cinecastui.qrc" >:/CineCastUi/Resources/HDD2.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="title" >
               <string>Disk Space(MB)</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7" >
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5" >
                 <item>
                  <widget class="QLabel" name="label" >
                   <property name="text" >
                    <string>Total:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2" >
                   <property name="text" >
                    <string>Avalible:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_5" >
                   <property name="text" >
                    <string>Used:</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6" >
                 <item>
                  <widget class="QLabel" name="label_3" >
                   <property name="text" >
                    <string>--</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_4" >
                   <property name="text" >
                    <string>--</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_6" >
                   <property name="text" >
                    <string>--</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeWidget" name="treeWidget_HDD_info" >
            <property name="font" >
             <font>
              <family>Book Antiqua</family>
              <pointsize>18</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="rootIsDecorated" >
             <bool>false</bool>
            </property>
            <column>
             <property name="text" >
              <string>Progress</string>
             </property>
            </column>
            <column>
             <property name="text" >
              <string>FilmName</string>
             </property>
            </column>
            <column>
             <property name="text" >
              <string>RecvTime</string>
             </property>
            </column>
            <column>
             <property name="text" >
              <string>FilmLength</string>
             </property>
            </column>
            <column>
             <property name="text" >
              <string>RecvSegment</string>
             </property>
            </column>
            <column>
             <property name="text" >
              <string>TotalSegment</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <spacer name="horizontalSpacer_6" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_ExporttoUSB" >
              <property name="minimumSize" >
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>120</width>
                <height>35</height>
               </size>
              </property>
              <property name="text" >
               <string>Export</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_13" >
              <property name="minimumSize" >
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="frameShape" >
               <enum>QFrame::Box</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="alignment" >
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_DeleteDir_HDD" >
              <property name="minimumSize" >
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>120</width>
                <height>35</height>
               </size>
              </property>
              <property name="text" >
               <string>Delete</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3" >
          <property name="maximumSize" >
           <size>
            <width>1</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" >
              <item>
               <widget class="QLabel" name="label_USB" >
                <property name="font" >
                 <font>
                  <pointsize>14</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text" >
                 <string/>
                </property>
                <property name="pixmap" >
                 <pixmap resource="cinecastui.qrc" >:/CineCastUi/Resources/usbdisk.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2" >
                <property name="title" >
                 <string>Disk Space(MB)</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_8" >
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_7" >
                   <item>
                    <widget class="QLabel" name="label_7" >
                     <property name="text" >
                      <string>Total:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_8" >
                     <property name="text" >
                      <string>Avalible:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_9" >
                     <property name="text" >
                      <string>Used:</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_8" >
                   <item>
                    <widget class="QLabel" name="label_10" >
                     <property name="text" >
                      <string>0</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_11" >
                     <property name="text" >
                      <string>0</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_12" >
                     <property name="text" >
                      <string>0</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeWidget" name="treeWidget_USB_info" >
            <property name="font" >
             <font>
              <family>Book Antiqua</family>
              <pointsize>18</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="rootIsDecorated" >
             <bool>false</bool>
            </property>
            <column>
             <property name="text" >
              <string>FilmName</string>
             </property>
            </column>
            <column>
             <property name="text" >
              <string>FilmLength</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" >
            <item>
             <spacer name="horizontalSpacer_10" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>13</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_Refush_Disk" >
              <property name="minimumSize" >
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>120</width>
                <height>35</height>
               </size>
              </property>
              <property name="text" >
               <string>Refush Disk</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_8" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>18</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_USB_UnMount" >
              <property name="minimumSize" >
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>120</width>
                <height>35</height>
               </size>
              </property>
              <property name="text" >
               <string>Unmount</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_9" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>18</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_DeleteDir_USB" >
              <property name="minimumSize" >
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>120</width>
                <height>35</height>
               </size>
              </property>
              <property name="text" >
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_11" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_CopyDir" >
        <property name="value" >
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Progress" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="cinecastui.qrc" />
 </resources>
 <connections/>
</ui>
